src/main/java/trivia/GameBetterWithLogging.java

```diff
@@ -94,4 +95,8 @@ private void printCurrentPlayerCoins() {
         + players.getCurrentPlayerCoins()
         + " Gold Coins.");
   }
+
+  private static boolean isOdd(int roll) {
```

Comment:
it is ok the extract the predicate, but naming it to illustrate business rule makes business logic easier to follow

Reply:
If the business rule is more complex than a single well-defined abstraction (like isOdd), then it makes sense to abstract it away, and if the reader is interested in the details they can navigate to the method. In this case it would just add unnecessary indirection.